app/decorators/post_search_result_set_decorator.rb

```diff
@@ -37,7 +37,7 @@ def to_og_params
     {
       locale: "ja_JP",
       type: "article",
-      title: to_meta_title,
+      title: "#{to_meta_title} | #{site.name}",
```

Comment:
Do you mean https://github.com/kpumuk/meta-tags#mirrored-values ?
It's reasonable. I'll use this feature.